docs/monitoring/tags.md

```diff
@@ -0,0 +1,12 @@
+title: Tags 
+description: A Tag is an attribute of a data point (or metric) that could be used for grouping and filtering.
+
+[Tags](../../tags) are sent by Sematext's Agents as part of every data point. Tags are shown in the UI as filters and group by elements. 
+
+For example, as part of Docker container metrics, the agent sends hostname on which the container is running, container identifier, container name, container image as tags which appear as filters in UI. User can then group or filter the container metrics using these tags.
```

Comment:
User can then group or filter the container metrics using these tags
=> 
User can then group or filter the container metrics using these tags as well as build custom charts.
(link build custom charts to the new CB page, if that makes sense)

Reply:
Fixed